web/templates/index.html

```diff
@@ -2,18 +2,46 @@
 {% load staticfiles %}
 
 {% block content %}
-<div id="main-content"></div>
+<!--<div id="main-content"></div> -->
+<style type="text/css">
```

Comment:
Agreed with what Guy said above, this should be in a separate css file, and that the style should be applied to a class rather than `body`.